elephant/spike_train_dissimilarity.py

```diff
@@ -35,8 +37,9 @@ def _create_matrix_from_indexed_function(
     return mat
 
 
-def victor_purpura_dist(
-        trains, q=1.0 * pq.Hz, kernel=None, sort=True, algorithm='fast'):
+@deprecated_alias(trains='spiketrains', q='cost_factor')
+def victor_purpura_dist(spiketrains, cost_factor=1.0 * pq.Hz, kernel=None,
```

Comment:
If also renaming functions, `_dist` should be `_distance`

Reply:
Indeed, this `dist` could be misread as distribution...